src/api/tokens/mod.rs

```diff
@@ -0,0 +1,38 @@
+// This file is part of rust-web/twig
+//
+// For the copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+/*!
+Tokens, received from lexer output.
+*/
```

Comment:
I read that multi-line comments are not rust-standard - but I can see the motivation to make life easier here. I plan to have a look at rustfmt. Maybe this can help with rust-like formatting and convert things like this? If not now - possibly in the future. :-)